featherpad/CMakeLists.txt

```diff
@@ -3,6 +3,9 @@ set(HUNSPELL_MINIMUM_VERSION "1.6")
 
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/cmake/Modules/")
 
+include_directories(/usr/local/include/)
+set(CMAKE_INSTALL_PREFIX "/Applications")
+
```

Comment:
`CMAKE_INSTALL_PREFIX` is already set to "/Applications" for APPLE in the main cmake file (compilation should be done in the main directory).

Reply:
Yeah, I agree. It *should* be so. However, it sets to `/usr/local/` without that line ..

Reply:
> However, it sets to /usr/local/ without that line ..

It really shouldn't: https://github.com/tsujan/FeatherPad/blob/6163f2724c79c4b7990d24ed1a27d188fcd5ceff/CMakeLists.txt#L9

Reply:
Maybe it's all about `CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT`?

Reply:
It's worth an investigation. I'll do it tomorrow.

Reply:
OK, why tomorrow?... If you use `mkdir build && cd build` and then `cmake ..` inside the main directory of a clean source, you should see `FeatherPad/build/CMakeCache.txt`, which includes something like this:

```
//Install path prefix, prepended onto install directories.
CMAKE_INSTALL_PREFIX:PATH=/usr/local
```

That line sets `CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT` to true and then, APPLE's installation path will be set to "/Applications".

If you don't see a line like that, there may be an issue in your cmake installation or, perhaps, cmake has a different behavior under macOS. In the second case, clean the build directory and explicitly do:

```
cmake .. -DCMAKE_INSTALL_PREFIX=/Applications
```

Reply:
I have this in my CMakeCache.txt
```
//Install path prefix, prepended onto install directories.
CMAKE_INSTALL_PREFIX:PATH=/usr/local
```

Reply:
So, nothing is different in this regard. The magic should happen somewhere else.